src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp

```diff
@@ -140,6 +140,54 @@ namespace OpenMS
     background = avg_noise_level * nr_points;
   }
 
+  void MRMTransitionGroupPicker::calculateBgEstimationExact_(const MSChromatogram& chromatogram,
+      double best_left, double best_right, double peak_height, double & background, double & avg_noise_level)
+  {
+    // determine (in the chromatogram) the intensity at the left / right border
+    double intensity_left = 0.0;
+    double rt_apex = 0.0;
+    double intensity_right = 0.0;
+    for (MSChromatogram::const_iterator it = chromatogram.begin() + 1; it != chromatogram.end(); ++it)
+    {
+      MSChromatogram::const_iterator it_prev = it;
+      --it_prev; //previous point
+
+      if (it->getMZ() >= best_left && it_prev->getMZ() < best_left)
+      {
+        intensity_left = it->getIntensity();
+      }
+      else if (it->getIntensity() >= peak_height && it_prev->getIntensity() < peak_height)
+      {
+        rt_apex = it->getMZ();
+      }
+      else if (it->getMZ() >= best_right && it_prev->getMZ() < best_right)
+      {
+        intensity_right = it->getIntensity();
+      }
+    }
+
+    double intensity_max, intensity_min, rt_min;
+    if (intensity_left >= intensity_right)
+    {
+      intensity_max = intensity_left;
+      intensity_min = intensity_right;
+      rt_min = best_right;
+    }
+    else 
+    {
+      intensity_max = intensity_right;
+      intensity_min = intensity_left;
+      rt_min = best_left;
+    }
+    double delta_int = intensity_max - intensity_min;
+    double delta_rt = best_right - best_left;
+    double delta_rt_apex = std::fabs(rt_min-rt_apex);
+    double delta_int_apex = delta_int*delta_rt_apex/delta_rt;
```

Comment:
as I said before, this will fail badly unless your spacing is exactly 1 second. You should use `double delta_int_apex = delta_int*delta_rt_apex/nr_points;`

see also the code above calculateBgEstimationAverage_

Reply:
If the trapezoid integration introduced in feature/peakAnnotation is now the default, should this not be changed?  I.e., would it be best to change the calculateBgEstimationAverage_ to match instead?

Reply:
trapezoid integration is currently just an experimental feature and not the default